### PR TITLE
Fix broken link in "API Priority and Fairness" page

### PR DESCRIPTION
**Pull request by mengqiy**
Fix a broken link that I introduced in https://github.com/kubernetes/website/pull/42075.

### CODE REVIEW

**Review by aj11anuj — APPROVED** [review]
Link is properly fixed and now working fine, 👍

**Comment by dipesh-rawat** [comment]
/retitle Fix broken link in "API Priority and Fairness" page

**Review by dipesh-rawat — COMMENTED** [review]
Thanks for the fix.
/lgtm

[Preview](https://deploy-preview-43064--kubernetes-io-main-staging.netlify.app/docs/concepts/cluster-administration/flow-control/#what-s-next)

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 1]
```diff
@@ -777,7 +777,7 @@ Example FlowSchema object to isolate list event requests:
 
 ## {{% heading "whatsnext" %}}
 
-- You can visit flow control [reference doc](/docs/reference/flow-control/) to learn more about troubleshooting.
+- You can visit flow control [reference doc](/docs/reference/debug-cluster/flow-control/) to learn more about troubleshooting.
```
Comment (kbhawkey) [inline content/en/docs/concepts/cluster-administration/flow-control.md:780]: Possibly edit the list item:
To learn more about troubleshooting, see the [Flow Control](/docs/reference/debug-cluster/flow-control/) reference.

Comment (windsonsea) [reply content/en/docs/concepts/cluster-administration/flow-control.md:780]: +1

**Comment by kbhawkey** [comment]
Thanks @mengqiy 
/approve

**Pull request merged**